walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -175,7 +177,7 @@ public void nestedEvents() {
   }
 
   @Test
-  public void doubleBookedPeople() {
+  public void query_doubleBookedPeople() {
```

Comment:
usually you title these unit tests as <method>_<conditions>_<expectedResults>. So for this unit test I think it should be named as: query_doubleBookedAttendees_<expectedResult> or is it query_noAvailableSlots_doubleBookedAttendees?

Reply:
I think your other unit tests also need to be renamed to follow this pattern.

Reply:
I updated the method names to include the number of available time slots.
Done!